specification/communication/data-plane/Identity/stable/2023-08-01/CommunicationIdentity.json

```diff
@@ -0,0 +1,460 @@
+        }
+      }
+    },
+    "CommunicationIdentityAccessToken": {
+      "description": "An access token.",
+      "required": [
+        "expiresOn",
+        "token"
+      ],
+      "type": "object",
+      "properties": {
+        "token": {
+          "description": "The access token issued for the identity.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "expiresOn": {
+          "format": "date-time",
+          "description": "The expiry time of the token.",
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationIdentityAccessTokenRequest": {
+      "required": [
+        "scopes"
+      ],
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "description": "List of scopes attached to the token.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationIdentityTokenScope"
+          }
+        },
+        "expiresInMinutes": {
+          "format": "int32",
+          "description": "Optional custom validity period of the token within [60,1440] minutes range. If not provided, the default value of 1440 minutes (24 hours) will be used.",
+          "type": "integer"
+        }
+      }
+    },
+    "CommunicationIdentityAccessTokenResult": {
+      "description": "A communication identity with access token.",
+      "required": [
+        "identity"
+      ],
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/CommunicationIdentity"
+        },
+        "accessToken": {
+          "$ref": "#/definitions/CommunicationIdentityAccessToken"
+        }
+      }
+    },
+    "CommunicationIdentityCreateRequest": {
+      "type": "object",
+      "properties": {
+        "createTokenWithScopes": {
+          "description": "Also create access token for the created identity.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationIdentityTokenScope"
+          }
+        },
+        "expiresInMinutes": {
+          "format": "int32",
+          "description": "Optional custom validity period of the token within [60,1440] minutes range. If not provided, the default value of 1440 minutes (24 hours) will be used.",
+          "type": "integer"
+        }
+      }
+    },
+    "CommunicationIdentityTokenScope": {
+      "description": "List of scopes for an access token.",
+      "enum": [
+        "chat",
+        "voip",
+        "email"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CommunicationIdentityTokenScope",
+        "modelAsString": true
+      }
+    },
+    "TeamsUserExchangeTokenRequest": {
+      "required": [
+        "appId",
+        "token",
+        "userId"
+      ],
+      "type": "object",
+      "properties": {
+        "token": {
+          "description": "Azure AD access token of a Teams User to acquire a new Communication Identity access token.",
+          "minLength": 1,
```

Comment:
this field is newly introduced in this PR in a couple of places, does it really makes sense to introduce it?

Reply:
Removed.